docs/README.md

```diff
@@ -80,7 +80,8 @@ This app exports all of the following blocks. You can see more detailed informat
   "summary-totalizers": {
     "props": {
       "showTotal": true,
-      "showDeliveryTotal": true
+      "showDeliveryTotal": true,
+      "showOriginalTotal": false
```

Comment:
I missed a description of what the prop means, but I couldn't find where to put it. Line 134 seems the place but it doesn't have the description of the other props.

Reply:
Updated the docs